src/common/accelcmn.cpp

```diff
@@ -112,6 +112,23 @@ static const struct wxKeyName
     { WXK_WINDOWS_RIGHT,    /*TRANSLATORS: Name of keyboard key*/ wxTRANSLATE("Windows_Right") },
     { WXK_WINDOWS_MENU,     /*TRANSLATORS: Name of keyboard key*/ wxTRANSLATE("Windows_Menu") },
     { WXK_COMMAND,          /*TRANSLATORS: Name of keyboard key*/ wxTRANSLATE("Command") },
+    { WXK_BROWSER_BACK,     /*TRANSLATORS: Name of keyboard key*/ wxTRANSLATE("Back") },
```

Comment:
This one definitely needs to be changed to something else, we can't have non unique strings here.

Actually I also wonder if we need to have these keys here at all, I don't think I've ever seen them explicitly listed as accelerators. The point of the special keys is that they have a defined function, so does it really make sense to list them here? Either they do what is written on them, in which case it's useless, or it doesn't, in which case it's very confusing and shouldn't be encouraged.